.storybook/locales/en.js

```diff
@@ -26,11 +26,23 @@ export default {
     },
     export: {
       title: 'Export',
-      description: 'You will receive an email containing a link that contains the export',
+      description: 'You will receive an email containing a link that contains the export.',
       filename: 'File name',
-      defaultFilename: 'My export',
+      defaultFilename: 'AGENT_NAME_OR_NODE_NAME - EXPORT_NAME',
```

Comment:
Totally agreed with you.
At first, I've been asked to auto generate this filename based on a node name/agent name + export name, but since this felt wrong to me, I proposed this change instead.